.buildkite/pipeline.yml

```diff
@@ -31,6 +31,7 @@ steps:
       - 'ln -s /var/lib/fc-ci/vmlinux.bin testdata/vmlinux'
       - 'ln -s /usr/local/bin/firecracker testdata/firecracker'
       - 'ln -s /usr/local/bin/jailer testdata/jailer'
-      - "make test EXTRAGOARGS='-v -count=1'"
+      - 'sudo chown 999:999 testdata/vmlinux'
+      - "sudo make test EXTRAGOARGS='-v -count=1'"
```

Comment:
Let's leave this step alone, and make a new one that is specifically for root tests (the way that I did [here](https://github.com/firecracker-microvm/firecracker-containerd/blob/83b8a18b81ae606fe4c6997021dd50c8438ef193/.buildkite/pipeline.yml#L25-L33)).

Reply:
Sounds good. Will move that into a root section